Configure.ps1

```diff
@@ -23,10 +23,10 @@
 # 6) Optionnaly if you want to cleanup the applications in the Azure AD, run:
 #      CleanUp $apps
 #    The applications are un-registered
+param([PSCredential]$Credential="", [string]$TenantId="")
```

Comment:
The Configure.ps1 script now has parameters (TenantId and Credentials). It can be ran from a continous integration build/Release